pages.py

```diff
@@ -69,15 +76,18 @@ def vars_for_template(self):
 
     def before_next_page(self):
         if self.timeout_happened:
+            self.player.player_timed_out += 1
```

Comment:
Test this to make sure it defaults to 0, so that it does not throw an error when you increment the first time.

Reply:
Done